__tests__/endpoints.test.js

```diff
@@ -96,3 +96,43 @@ describe("/api", () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('test for 200 status on completion', () => {
+        return request(app).get(`/api/articles/3`).expect(200);
+    });
+    test('should have the correct article by id', () => {
+        return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+            expect(result.body).toEqual([{
```

Comment:
Here you are expecting an array back. How many reviews would you always expect to receive back with this endpoint? Is it necessary to send back a data type which allows multiple objects? 

On another unrelated note - The assertion made here is that you expect this _exact_ object to come back. Later if you add any columns to the database this test would fail as it's brittle. 
Better practice would entail separate assertions for each property. Ie expect (result.body).toHaveProperty(article_id) && using typeof to assert the data type of each property. 
There is a later ticket that requires you to add an additional column to the database and it would be ideal if you didn't have to go back and make amendments to this test and any others.